CONTRIBUTING.md

```diff
@@ -87,30 +77,8 @@ sure to post a comment after pushing. The new commits will show up in the pull
 request automatically, but the reviewers will not be notified unless you
 comment.
 
-Before the pull request is merged, make sure that you squash your commits into
```

Comment:
Squashing is not necessary anymore and for most people it breaks there branch as they are not experienced with git